keyloop/api/v1/permission_grant.py

```diff
@@ -22,7 +22,7 @@ def __acl__(self):
 
 class CollectionPostSchema(marshmallow.Schema):
     path = marshmallow.fields.Nested(BasePathSchema)
-    body = marshmallow.fields.Nested(PermissionGrantSchema)
+    body = marshmallow.fields.Nested(PermissionGrantSchema())
```

Comment:
Pq esses parenteses? Deixar a surgestão aqui:
```suggestion
    body = marshmallow.fields.Nested(PermissionGrantSchema)
```